app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -8,349 +8,320 @@
-                        android:layout_height="wrap_content"
-                        android:scaleType="centerCrop"
-                        android:src="@drawable/robert_downey" />
-                </androidx.cardview.widget.CardView>
-
-                <TextView
-                    style="@style/artist_style"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:text="@string/robert_downey_text" />
-            </LinearLayout>
-
-            <LinearLayout
-                android:id="@+id/second_artist"
-                android:layout_width="0dp"
-                android:layout_height="118dp"
-                android:layout_weight="1"
-                android:layout_marginTop="7dp"
-                android:layout_marginEnd="8dp"
-                android:orientation="vertical">
-
-                <androidx.cardview.widget.CardView
-                    android:layout_width="wrap_content"
-                    android:layout_height="80dp"
-                    app:cardCornerRadius="4dp">
-
-                    <ImageView
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:scaleType="centerCrop"
-                        android:src="@drawable/chris_evans" />
-                </androidx.cardview.widget.CardView>
-
-                <TextView
-                    style="@style/artist_style"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:text="@string/chris_evans" />
-            </LinearLayout>
-
-            <LinearLayout
-                android:id="@+id/third_artist"
-                android:layout_width="0dp"
-                android:layout_height="118dp"
-                android:layout_weight="1"
-                android:layout_marginTop="7dp"
-                android:layout_marginEnd="8dp"
-                android:orientation="vertical">
+            android:text="@string/cast_text"
+            app:layout_constraintLeft_toRightOf="@id/guideline_margin_start"
+            app:layout_constraintRight_toLeftOf="@id/guideline_margin_end"
+            app:layout_constraintTop_toBottomOf="@id/tv_story_line_details" />
 
-                <androidx.cardview.widget.CardView
-                    android:layout_width="wrap_content"
-                    android:layout_height="80dp"
-                    app:cardCornerRadius="4dp">
-
-                    <ImageView
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:scaleType="centerCrop"
-                        android:src="@drawable/mark_ruffalo" />
-                </androidx.cardview.widget.CardView>
+        <com.google.android.material.imageview.ShapeableImageView
```

Comment:
I believe you can implement it closer to target design.
![image](https://user-images.githubusercontent.com/1586158/99889820-93c09180-2c0d-11eb-8f28-a452123d3739.png)

Reply:
btw can you make it full screen without that status bar on top?